### PR TITLE
Update comment formatting for "Request changes"

### PR DESCRIPTION
**Pull request by cadeParade**
## Description
This PR updates comment thread formatting to reflect the new "request changes" state. 
It shows a solid orange badge on comment threads that were "changes requested" on the current build, and a white background badge for comment threads that were "changes requested previously". 
"Changes requested previously" badges also have a link to the snapshot where changes were originally requested.

## Reviewing this PR

_**Is Covered by Automated Tests?: YES**_

### CODE REVIEW

File: app/components/collaboration/collaboration-comment.js [hunk 1]
```diff
@@ -21,6 +21,17 @@ export default Component.extend({
     return moment(this.comment.createdAt).fromNow();
   }),
 
+  areChangesRequestedPreviously: computed(
```
Comment (unicornzero) [inline app/components/collaboration/collaboration-comment.js]: This function appears to verify if the current `snapshot` is the one that started the `commentThread`, so it feels weird to have `ChangesRequested` in the function name. How about something like `isOriginatingSnapshot`?

File: tests/acceptance/build-test.js [hunk 2]
```diff
@@ -263,28 +256,44 @@ describe('Acceptance: Build', function() {
...
       expect(secondSnapshot.commentThreads.length).to.equal(1);
       expect(secondSnapshot.header.numOpenCommentThreads).to.equal('1');
+      expect(secondSnapshot.collaborationPanel.commentThreads[0].isRejectBadgeVisible).to.equal(
+        true,
+      );
+      expect(secondSnapshot.collaborationPanel.commentThreads[0].wasRejectedPreviously).to.equal(
+        false,
+      );
 
+      // TODO expect snapshot to be rejected
```
Comment (unicornzero) [inline tests/acceptance/build-test.js]: Leftover comment?

File: yarn.lock [hunk 3]
```diff
@@ -6817,7 +6824,7 @@ ember-percy@percy/ember-percy.git#agentified:
   version "1.4.1"
   resolved "https://codeload.github.com/percy/ember-percy/tar.gz/6ff0656b7a2283b39289688683975036a1931ea9"
   dependencies:
-    "@percy/agent" "^0.7.2"
+    "@percy/agent" "^0.1.2"
```
Comment (unicornzero) [inline yarn.lock]: Were the changes to this file intentional?

Comment (cadeParade) [reply yarn.lock]: Not this line! Thanks for catching.

File: tests/acceptance/build-test.js (hunk 2 above)
Comment (cadeParade) [reply tests/acceptance/build-test.js]: fixed

File: app/components/collaboration/collaboration-comment.js (hunk 1 above)
Comment (cadeParade) [reply app/components/collaboration/collaboration-comment.js]: renamed

**Pull request merged**